jwst/datamodels/schemas/slitmeta.schema.yaml

```diff
@@ -79,6 +79,18 @@ properties:
     default: 0.0
     fits_keyword: SRCYPOS
     fits_hdu: SCI
+  extr_x:
```

Comment:
These keywords will only ever appear in extracted (x1d) products, hence they don't need to be in this schema. This schema is only used for pre-extraction products.

Reply:
Ok thanks for the clarification. I was misunderstanding what the slitmeta schema was